docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -0,0 +1,99 @@
+= Parameters
+
+The parent key for all of the following parameters is `pkg.monitoring`.
+
+== `images`
+
+[horizontal]
+type:: dictionary
+
+Dictionary containing the container images directly used by this package.
+The dictionary doesn't contain images managed by included components
+
+
+== `prometheus`
+
+
+=== `prometheus.ingress.enabled`
+
+[horizontal]
+type:: boolean
+default:: `true`
+
+Whether to deploy an ingress for Prometheus.
```

Comment:
Maybe it should be mentioned that this is always pointing to the oauth2 proxy?

Reply:
Hmm good point. I'll add something for that